src/java/org/apache/cassandra/db/compaction/CompactionTask.java

```diff
@@ -244,7 +247,8 @@ public boolean apply(SSTableReader sstable)
             for (int i = 0; i < mergedRowCounts.length; i++)
                 totalSourceRows += mergedRowCounts[i] * (i + 1);
 
-            String mergeSummary = updateCompactionHistory(cfs.keyspace.getName(), cfs.getTableName(), mergedRowCounts, startsize, endsize, compactionType.type);
+            String mergeSummary = updateCompactionHistory(cfs.keyspace.getName(), cfs.getTableName(), mergedRowCounts, startsize, endsize,
+                ImmutableMap.of(CompactionHistoryProperty.COMPACTION_PROPERTIES_KEYS[0], compactionType.type));
```

Comment:
This is a good place to use a constant for compaction type property name

Reply:
+1